spec/lib/process_settings_spec.rb

```diff
@@ -12,7 +12,6 @@
 
     describe 'when lazy loading' do
       before do
-        expect(ActiveSupport::Deprecation).to receive(:warn).with("lazy creation of Monitor instance is deprecated and will be removed from ProcessSettings 1.0")
```

Comment:
Shouldn't this test still warn because `instance` will call `default_instance`, where the warning is now.

Reply:
Agreed!

Reply:
I've gone ahead and added the expectation back.  This test is now showing that we are going to get a deprecation warning from the lazy loading of `Monitor.instance` which already calls out to switch to setting the monitor instance on `ProcessSettings`.